openapi/ga4gh-tool-discovery.yaml

```diff
@@ -391,6 +394,13 @@ paths:
           description: >-
             An identifier of the tool version for this particular tool registry,
             for example `v1`.
+        - name: format
+          in: query
+          required: false
+          type: string
+          description: specify format=zip to download a zip file of all files available
+          enum:
+            - zip
       responses:
         '200':
```

Comment:
Do you need to add the response for zip files like you did for OpenAPI, or does Swagger not support that?

Reply:
See https://swagger.io/docs/specification/2-0/describing-responses/ FAQ